sql-cloudant/README.md

```diff
@@ -62,6 +62,8 @@ cloudant.protocol|https|protocol to use to transfer data: http or https
 cloudant.host||cloudant host url
 cloudant.username||cloudant userid
 cloudant.password||cloudant password
+cloudant.useQuery|false|When enabled, for query not using index or view, _find will be used instead of _all_docs, some query predicates will be driven into datastore. However, RDD partition is ONE during _find, so parallel loading is not achieved
```

Comment:
This description is a bit unclear.  How about: `When set to true, the _find endpoint will be queried.  The _all_docs endpoint will be used when set to false (default setting).  Note: When querying against the _find endpoint, RDD partition is limited to one so parallel loading is not an available option.`  Also, I'm having a hard time understanding this: `some query predicates will be driven into datastore`.